{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -24,10 +24,6 @@ classifiers = [
 dependencies = [
 ]
 description = "{{ cookiecutter.project_short_description }}"
-ignore = [
-    "D203", # no-blank-line-before-class
-    "D212", # multi-line-summary-first-line
```

Comment:
These two both clash with another `D` rule, so one of them has to be commented out. Sure we can leave it to the user but won't that add confusion/assumed knowledge?

Reply:
My er.. claim is that the whole list is in the wrong place. Shouldn't it be under a `[tool.pydocstyle]` or somesuch?

Reply:
Ah yes, it should be like this
https://github.com/paddyroddy/python-template/blob/563c799392ec3302e035d904ffb562eda8318a6c/pyproject.toml#L73C1-L79